### ISSUE TITLE
dapple --help not showing options

### ISSUE THREAD

**Issue by ryepdx** [nexusdev/dapple#257]
There are a bunch of cross-command options not being shown in dapple's help output. (For example, the `--subpackages` flag.)

**Comment by mhhf** [comment]
are you sure about this? They aren'd rendered as colorful as the commands but i see them appearing at the bottom:
```
$ dapple --help
dapple version: 0.7.2-c4f35b2b46

USAGE: dapple COMMAND [OPTIONS]

COMMANDS:

[...]
        
    dapple chain server                 start an rpc server on the current branch
        

OPTIONS:
    -e ENVIRONMENT, --environment ENVIRONMENT [default: evm]
     -c DAPPLERC, --config DAPPLERC
     -s, --subpackages
     --report
     --version
     --optimize
```

**Comment by ryepdx** [comment]
This is what I see:

```
(nexus) dev@455085490f74:~/devenv/dapple$ dapple help
Usage:
    dapple --help 
    dapple config 
    dapple init 
    dapple doctor 
    dapple install [--save] [options] [<package> <url-or-version>]
    dapple build [-r <RegExp>] [--global=<globalVar>] [--template=<templateName>] [options] [--no-deploy-data] [--tests]
    dapple test [-r <RegExp>] [options] [--skip-build]
    dapple new test <class>
    dapple run <script> [--force] [--no-simulation] [options]
    dapple step <string> [options]
    dapple publish [options]
    dapple add <path>
    dapple ignore <path>
(nexus) dev@455085490f74:~/devenv/dapple$ 
```

**Comment by mhhf** [comment]
wow! `dapple --help` should work, very interesting.
How did you install dapple?

```
$ dapple --help
dapple version: 0.7.2-c4f35b2b46
```
dapple help tries to read the local `.git/ORIG_HEAD` to print a clear commit hash along the current version. I assume there is no `.git` repo along your installation. 
I thing I caused this fuckup, don't thinking that installing dapple via npm don't provide the `.git` folder.

I'll fix it asap in the next version!

**Comment by mhhf** [comment]
should work in v0.8, should be revisited and closed with the #256 merge
